editor/inspector/assets/fbx/preview.js

```diff
@@ -679,16 +679,16 @@ exports.methods = {
 
     addAssetChangeListener(add = true) {
         if (!add && this.hasListenAssetsChange) {
-            Editor.Message.removeBroadcastListener('asset-db:asset-change', this.onAssetChangeBind);
+            Editor.Message.__protected__.removeBroadcastListener('scene:all-asset-changed', this.onAssetChangeBind);
```

Comment:
This has a significant impact, which means that the asset change monitoring by the inspector has shifted from the db to the scene.

Reply:
its ok, when asset changed,scene update asset,then preview get messge to update preview